backend/ipmi.pm

```diff
@@ -17,7 +17,8 @@ sub new ($class) {
 
 sub ipmi_cmdline ($self) {
     $bmwqemu::vars{"IPMI_$_"} or die "Need variable IPMI_$_" for qw(HOSTNAME USER PASSWORD);
-    return ('ipmitool', '-I', 'lanplus', '-H', $bmwqemu::vars{IPMI_HOSTNAME}, '-U', $bmwqemu::vars{IPMI_USER}, '-P', $bmwqemu::vars{IPMI_PASSWORD});
+    my $ipmi_options = $bmwqemu::vars{"IPMI_OPTIONS"} // '-I lanplus';
```

Comment:
```suggestion
    my $ipmi_options = $bmwqemu::vars{IPMI_OPTIONS} // '-I lanplus';
```
but perlcritic should complain about that as well

Reply:
hmm: https://github.com/os-autoinst/os-autoinst/actions/runs/11796830598/job/32859558179?pr=2570#step:4:89
```
5: Policy "Perl::Critic::Policy::OpenQA::HashKeyQuotes" is not installed.
89
5: Policy "Perl::Critic::Policy::OpenQA::RedundantStrictWarning" is not installed.
```

Reply:
https://progress.opensuse.org/issues/155188 "perltidy&perlcritic in openQA"
vs.
https://progress.opensuse.org/issues/155191 "perlcritic in os-autoinst"
It would really be good to not have so much time between similar tasks

Reply:
done> but perlcritic should complain about that as well

but it doesn't. How can we make it complain?

Reply:
by working on https://progress.opensuse.org/issues/155191 :)